checkstyle-tester/diff.groovy

```diff
@@ -340,23 +343,20 @@ def getCommitSha(commitId, repoType, srcDestinationDir) {
     return sha.replace('\n', '')
 }
 
-def getCloneCmd(repoType, repoUrl, srcDestinationDir) {
-    def cloneCmd = ''
-    switch (repoType) {
-        case 'git':
-            cloneCmd = "git clone $repoUrl $srcDestinationDir"
-            break
-        default:
-            throw new IllegalArgumentException("Error! Unknown $repoType repository.")
-    }
-    return cloneCmd
-}
-
 def cloneRepository(repoName, repoType, repoUrl, commitId, srcDir) {
+    println "In clone repository"
     def srcDestinationDir = getOsSpecificPath("$srcDir", "$repoName")
     if (!Files.exists(Paths.get(srcDestinationDir))) {
-        def cloneCmd = getCloneCmd(repoType, repoUrl, srcDestinationDir)
+        def cloneCmd
```

Comment:
We try to stick to java conventions when possible:
```suggestion
        final String cloneCmd
```

Please do this wherever you can.

Reply:
Done